javascript/javascript1/week3/travel-time-calculation.js

```diff
@@ -0,0 +1,18 @@
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432,
+};
+
+function timeCalculate(info) {
+  return info.destinationDistance / info.speed; //formula: time = distance / speed
+}
+
+const travelTime = timeCalculate(travelInformation);
+let hours = Math.floor(travelTime);
+let minutes = Math.ceil((travelTime % Math.floor(travelTime)) * 100);
```

Comment:
- `Math.floor(travelTime)` is `hours` so you can use `hours` instead.
- why * `100` I think we multiply with `60` to convert
- minutes could be:
```js
const minutes = Math.round((totalTime - hours) * 60);
```
or 
```js
const minutes = Math.round((totalTime % hours) * 60);
```